C# extension cannot start omnisharp on Linux due to wrong casing of 'omnisharp' executable

1. Install daily build of .NET CLI
2. Install the 1.0.0-rc2 build of the C# extension

Result:
[INFO] Starting OmniSharp at '/home/greggm/clicon'...
[ERROR] Error: Could not fnd launch file in /home/greggm/.vscode-alpha/extensions/ms-vscode.csharp-1.0.0-rc2/.omnisharp. Expected 'run', 'omnisharp', 'omnisharp'

Expected:
Omnisharp starts

This is from:

    omnisharp-vscode\src\omnisharpPath.ts(14):const omnisharpExeFileName = process.platform === 'win32' ? 'omnisharp.exe' : 'omnisharp';
    omnisharp-vscode\src\omnisharpPath.ts(41):        throw new Error(`Could not fnd launch file in ${filePathOrFolder}. Expected '${runFileName}', '${omnisharpFileName}', '${omnisharpExeFileName}'`);

The issue is that the file casing is wrong.

--- a/src/omnisharpPath.ts
+++ b/src/omnisharpPath.ts
@@ -10,8 +10,8 @@ import * as path from 'path';
 import * as vscode from 'vscode';
 
 const runFileName = process.platform === 'win32' ? 'run.cmd' : 'run';
-const omnisharpFileName = process.platform === 'win32' ? 'omnisharp.cmd' : 'omnisharp';
-const omnisharpExeFileName = process.platform === 'win32' ? 'omnisharp.exe' : 'omnisharp';
+const omnisharpFileName = process.platform === 'win32' ? 'OmniSharp.cmd' : 'OmniSharp';
+const omnisharpExeFileName = process.platform === 'win32' ? 'OmniSharp.exe' : 'OmniSharp';
 
 function getLaunchFilePath(filePathOrFolder: string): Promise<string> {
     return fs.lstatAsync(filePathOrFolder).then(stats => {

